_maps/configs/pubby.json

```diff
@@ -1,15 +1,22 @@
 {
 	"map_name": "Pubby-class Light Carrier",
+	"prefix": "ISV",
+	"namelists": ["GENERAL", "BRITISH_NAVY", "MYTHOLOGICAL", "WEAPONS"],
 	"map_short_name": "Punny-class",
```

Comment:
"map_short_name": "Pubby-class",

while amusing, the shortname actually shows up in the ship purchase menu and a few other places and should actually match the ship's name.

Reply:
oh,pft i didnt set it to be like that. it's always been that way